src/engines/witcher/lua/bindings.cpp

```diff
@@ -142,11 +142,17 @@ void LuaBindings::registerBindings() {
 	LuaScriptMan.endRegister();
 }
 
+CAuroraSettingsData g_auroraSettings;
```

Comment:
Does this have to be a *global* global? Can it be a static within `LuaBindings::luaGetAuroraSettings()` instead?